### PR TITLE
Fix breaking Ruby tests by adding new spec projects.

### PR DESCRIPTION
**Pull request by tirodkar**
Fix ruby test breaks caused by the Example.xcodeproj in gem/fixture/specs not having a Products/ group within them. This is fixed by adding a more fleshed out project.

### CODE REVIEW

**Comment by bootstraponline** [comment]
The Ruby build is still failing. I recommend disabling the `Style/VariableNumber` rubocop rule. It's a new rule they added. Following their recommendation results in less readable code.

**Comment by tirodkar** [comment]
Let's not disable it. I'll fix the warnings. Thanks!

**Comment by bootstraponline** [comment]
> Let's not disable it. I'll fix the warnings. Thanks!

Which is more readable:

- `swift22sha1`
- `swift_2_2_sha1`

Rubocop has some silly defaults which is why we're already disabling 8 rules.

**Comment by tirodkar** [comment]
@bootstraponline Done. Thanks.

File: .rubocop.yml [hunk 1]
```diff
@@ -35,3 +35,5 @@ Metrics/ModuleLength:
   Enabled: false
 Style/GuardClause:
   Enabled: false
+Style/VariableName:
+  Enabled:false
```
Comment (bootstraponline) [inline .rubocop.yml]: `Enabled:false` should be `Enabled: false`

**Comment by tirodkar** [comment]
Ruby tests look all green now.

**Pull request merged**